portal/src/error/fileUplaod.ts

```diff
@@ -0,0 +1,8 @@
+export interface APIFileUploadTooLargeError {
```

Comment:
Put this in `portal/src/error/resources.ts`. Remember the name and the reason should have changed.